reckless_lib/Cargo.toml

```
@@ -10,3 +10,4 @@ git2 = "0.15.0"
 log = "0.4.17"
 env_logger = "0.9.3"
 glob = "0.3.0"
+tokio = { version = "1.21.2", features = ["full"] }
```

```suggestion
tokio = { version = "1.21.2", features = ["process"] }
```

mh not sure that this is correct, we need only the process feature, you can find the docs here https://docs.rs/tokio/latest/tokio/#feature-flags

---

Yes, I should have been careful, not just copy blindly :smile: